docs/source/implementations/labeling_fixed_time_horizon.rst

```diff
@@ -0,0 +1,68 @@
+.. _implementations-labeling_fixed_time_horizon:
+
+====================
+Fixed Horizon Method
+====================
+
+Fixed horizon labels is a classification labeling technique used in the following paper: `Dixon, M., Klabjan, D. and
+Bang, J., 2016. Classification-based Financial Markets Prediction using Deep Neural Networks.
+arXiv:1603.08604. <https://arxiv.org/abs/1603.08604>`_
+
+Fixed time horizon is a common method used in labeling financial data, usually applied on time bars. The forward rate of return relative
+to :math:`t_0` over time horizon :math:`h` is calculated as follows:
+
+.. math::
+    r_{t0,t1} = \frac{p_{t1}}{p_{t0}} - 1
+
+Where :math:`t_1 = t_0 + h` is the time bar index after a fixed horizon of :math:`h` ticks have passed, and :math:`p_{t0}, p_{t1}`
+are prices at times :math:`t_0, t_1`. This method assigns a label based on comparison of rate of return to a threshold :math:`\tau`
+
+.. math::
+    -1 \text{ if } r_{t0, t1} < -\tau\\
+    0 \text{ if } -\tau \leq r_{t0, t1} \geq \tau\\
+    1 \text{ if } r_{t0, t1} > \tau
+
+Though time bars are the most common format for financial data, there can be potential problems with over reliance on time bars. First,
```

Comment:
> over reliance

to

> over-reliance 

Also, for style purposes, please either remove `First, ...`, or add later `Second, ...`